ToDo/ViewModels/MainWindowViewModel.cs

```diff
@@ -77,14 +83,13 @@ private void AddNewTodo()
 
                 _todoItemService.WriteTodos(TodoItems.Select(vm => vm.TodoItem));
 
+                NewTodoName = NEW_TODO;
```

Comment:
Beim Anwendungsstart, wird hier aber der Text nicht initial eingetragen.
Kannst du das bitte in meinem Code noch ergänzen.